diff -urNp -x '*.orig' fcron-3.1.2.org/Makefile.in fcron-3.1.2/Makefile.in
--- fcron-3.1.2.org/Makefile.in	2013-01-13 14:20:31.000000000 +0100
+++ fcron-3.1.2/Makefile.in	2023-02-02 08:58:38.547843071 +0100
@@ -137,7 +137,7 @@ initscripts:
 documentation:
 	$(MAKE) -C doc doc-if-none
 
-install: install-staged strip perms 
+install: install-staged strip
 ifeq ($(BOOTINSTALL), 1)
 	$(SRCDIR)/script/boot-install "$(INSTALL) -o $(ROOTNAME)" $(DESTSBIN) $(DEBUG) $(FCRONTABS) $(ANSWERALL) $(SRCDIR)
 endif
